package.json

```diff
@@ -29,39 +29,40 @@
 		"bump": "cliff-jumper",
 		"check-update": "cliff-jumper --dry-run",
 		"postinstall": "husky install .github/husky",
-		"prepack": "rollup-type-bundler -e node:util && pinst --disable",
+		"prepack": "rollup-type-bundler -e util && pinst --disable",
 		"postpack": "pinst --enable"
 	},
 	"devDependencies": {
 		"@commitlint/cli": "^17.0.3",
 		"@commitlint/config-conventional": "^17.0.3",
+		"@esbuild-plugins/node-modules-polyfill": "^0.1.4",
```

Comment:
Should maybe try `brode` instead https://github.com/geut/brode/tree/main/packages/esbuild-plugin-brode
It has support for the `node:` protocol, so you wouldn't even have to touch any imports.

Reply:
as discussed in discord, it's not working in windows so windows users (contributors) won't be able to build it. Also after trying in wsl even using 
```js

			injectGlobal: {
				process: false,
				buffer: false,
				assert: false,
				crypto: false,
				console: false,
				constants: false,
				events: false,
				fs: false,
				module: false,
				path: false,
				os: false,
				url: false,
				util: true,
				timers: false,
				tty: false,
				stream: false,
				zlib: false
			}
```
it's still bundling all the core modules and making the browser build 10k+ loc. it's also adding these module imports in cjs and mjs files. I'm making this draft now and will try some other alternatives

Reply:
For windows compatibility best I can tell is to implement `environmentsFilter` which defaults to a noop but that doesn't change the bundle size problem. Using `@esbuild-plugins/node-modules-polyfill` the `index.global.js` is 107KB compared to the 77KB of `index.js` and `index.mjs`, a 20 KB uptick to bundle `fast-deep-equal`, `lodash.uniqwith` and `util.inspect` is pretty acceptable.

Reply:
Yeah I can see that now, interesting. I'll see if something can be done about this some time.
Would be nice to not have to remove the `node:` protocol in source files for example.

Reply:
I've created a pr to support `node:` protocol https://github.com/remorses/esbuild-plugins/pull/20 but idk if that repo is still maintained. So I tried https://github.com/imranbarbhuiya/esbuild-plugins-node-modules-polyfill but for some reason it is saying that `inspect` isn't exported from `util` package 😞

Reply:
fixed in https://github.com/sapphiredev/shapeshift/pull/150/commits/47cd7c25fc11115631b9d01dda1482d175f00dc0 with a patch in the package